src/main/java/com/yodle/vantage/config/ExternalNeo4jConfig.java

```diff
@@ -31,8 +33,11 @@
     @Value("${neo4j.user:neo4j}") String neo4jUsername;
     @Value("${neo4j.user:password}") String neo4jPassword;
 
+    private static final Logger l = LoggerFactory.getLogger(ExternalNeo4jConfig.class);
+
     @Bean
     public DataSource neo4jDataSource() {
+        l.info("Connecting to neo4j server at {}:{}", neo4jServer, neo4jPort);
```

Comment:
when i initially started debugging this, it was hard to tell if i was running in embedded or external mode, so i added a bit of additional logging on startup.